setup/docker-compose.tests.yml

```diff
@@ -32,6 +32,7 @@ services:
     networks:
        - emission
 
+
```

Comment:
nit: extraneous whitespace

Reply:
This appears to be a very old version of the file, since I moved the compose to the integrationTests folder

Reply:
so revert the whitespace change?

Reply:
Definitely thought this file was something else! Reverted the change :)